react/src/App.jsx

```diff
@@ -78,6 +78,9 @@ function App() {
     window.Helpshift("updateHelpshiftConfig");
   };
 
+  // @NOTE: In order to add an event to the window, you should add it.
+  // When the component mounts using the useEffect hook
+  // and remove it when the component unmounts.
```

Comment:
Check grammar here.

`...you should add it when the component...`
instead of

`"you should add it.
When the component"`